sql/src/test/java/io/crate/analyze/CompoundLiteralTest.java

```diff
@@ -190,23 +116,21 @@ public void testArrayLiteral() throws Exception {
 
     @Test
     public void testArrayLiteralWithParameter() throws Exception {
```

Comment:
pls rename, since this is now no literal anymore, check also the other test names